ocaml/xapi/xapi_host.ml

```diff
@@ -1527,7 +1527,15 @@ let reset_networking ~__context ~host =
                 Db.PIF_metrics.destroy ~__context ~self:metrics
               end;
               Db.PIF.destroy ~__context ~self;
-            ) local_pifs
+  ) local_pifs;
+  List.iter (fun self -> 
```

Comment:
Why not follow the styles above that cleanup the `bonds`, `vlans` and `tunnels`? It could make the code easier to understand and maintain.

Reply:
I don't think that the existing code is particularly clear but I'm happy to change it.